inet/vibe/inet/url.d

```diff
@@ -220,21 +289,6 @@ struct URL {
 	{
 		return URL(url_string);
 	}
-	
-	/**
-	* Parse a 'plain' string into an `URL`.
-	*
-	* Unlike `URL.parse`, this expects its argument to be a 
-	* plain url (not percent-encoded nor punyencoded), and thus can contain
-	* non-ASCII characters as well as reserved ones (e.g. a space).
-	*
-	* Params:
-	*   url_string = A plaintext URL, which will be percent-encoded in the result.
-	*/
-	static URL parsePlain(string url_string)
```

Comment:
Isn't that breaking code ? Or was it not part of a release ?

Reply:
This was not yet released.